code-challenges.test.js

```diff
@@ -60,8 +98,197 @@ var hand2 = [5, 5, 3, 3, 4]
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 
+describe("When isFullHouse is given an array of 5 numbers,", () => {
+
+    it("returns a boolean value determining if the array is a full house or not.", () => {
 
+        const actResult = isFullHouse(hand1);
+        
+        expect(actResult).toEqual(true);
+    })
+    it("returns a boolean value determining if the array is a full house or not.", () => {
 
+        const actResult = isFullHouse(hand2);
+        
+        expect(actResult).toEqual(false);
+    })
+    it("returns a boolean value determining if the array is a full house or not.", () => {
 
+        const actResult = isFullHouse(hand3);
+        
+        expect(actResult).toEqual(false);
+    })
+})
 
 // b) Create the function that makes the test pass.
+
+// Deven + Sarah discussion on how to go about this problem:
+// check for unique values and the count of the values 
+// store the cards in an object (key:value pair)
+// find two unique values and check to see if there are multiple of them 
+// keeping track of the characters and tracking them: update the value within the object
+
+const isFullHouse = (array) => {
+
+    if(array.length !== 5){
```

Comment:
YES! Validating inputs! and "early returning" if it's invalid. Definitely keep this up! 💯